chart/kubeapps/templates/_helpers.tpl

```diff
@@ -239,10 +239,15 @@ Returns a JSON list of cluster names only (without sensitive tokens etc.)
 {{- end -}}
 
 {{/*
-Returns the name of the globalRepos namespace
+Returns the name of the global packaging namespace for the Helm plugin.
+It uses the value passed in the plugin's config, but falls back to the "release namespace + suffix" formula.
 */}}
-{{- define "kubeapps.globalReposNamespace" -}}
-{{- printf "%s%s" .Release.Namespace .Values.apprepository.globalReposNamespaceSuffix -}}
+{{- define "kubeapps.helmGlobalPackagingNamespace" -}}
```

Comment:
We were already using this helper function to retrieve the namespace, so I've changed it to: "use the globalPackagingNamespace in the plugin or fall back to the previous behavior"